docs/topics/api/addons.rst

```diff
@@ -5,18 +5,16 @@ Add-ons
 .. note::
 
     These APIs are experimental and are currently being worked on. Endpoints
-    may change without warning.
-
-    At the moment these APIs don't yet require authentication and therefore
-    are limited to listed public add-ons.
+    may change without warning. The only authentication method available at
+    the moment is :ref:`the internal one<api-auth-internal>`.
 
 ------
 Search
 ------
 
 .. _addon-search:
 
-This endpoint allows you to search through public add-ons.
+This endpoint allows you to search through public, listed add-ons.
```

Comment:
I'm not sure `, listed` provides any more context here. It makes it sounds incomplete to me.